app/src/main/java/app/revanced/manager/network/api/ReVancedAPI.kt

```diff
@@ -18,6 +15,8 @@ class ReVancedAPI(
 
     suspend fun getRelease(name: String) = service.getRelease(apiUrl(), name).transform { it.release }
```

Comment:
It would make more sense to call this `getLatestRelease` now that `getReleases` exists